app/views/posts/show.html.erb

```diff
@@ -8,11 +8,22 @@
   
   <h2><%= @post.title %></h2>
   <p><strong>Author:</strong> <%= @post.author.name %></p>
-  
+  <p><strong>Likes:</strong> <%= @post.likes.count %></p>
+  <p><strong>Comments:</strong> <%= @post.comments.count %></p>
+
   <p><strong>Title:</strong> <%= @post.title %></p>
   <p><strong>Text:</strong> <%= @post.text %></p>
-  <p><strong>Likes:</strong> <%= @post.likes_counter %></p>
-  <p><strong>Comments:</strong> <%= @post.comments_counter %></p>
-  
+
+  <h3>Comments:</h3>
+  <ul>
+    <% @post.comments.each do |comment| %>
+      <li>
+       <p><strong>Username:</strong> <%= comment.user.name %></p>
+        <%= comment.text %>
+       
+      </li>
+    <% end %>
+  </ul>
```

Comment:
- [ ] Great job ✨ Jus a little bit more to go. Kindly make sure to follow the given wireframe for styling as per [this project requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Implement%20the%20design%20from%20the%20sneak%20peek%20wireframes.)
- [ ] When I am navigating from the post index page, the user's name is present in the URL. kindly make sure that in the URL we have the id of the user not their name ⛏🤓

Current | Wireframe
---| ---|
![image](https://github.com/Kidd254/BlogApp/assets/106119105/d4e7cf38-9f35-406a-9f56-26956d7c692c) | ![image](https://github.com/Kidd254/BlogApp/assets/106119105/ad8bee20-3dbc-4968-ae28-c81d07e979f5)


- [ ] However, when I'm manually changing the URL, I am getting posts of Tom for Lilly also and vice versa. Please have a look at the following video 👇 Kindly fix this bug 🐛

https://github.com/Kidd254/BlogApp/assets/106119105/70d3597a-a93e-4ba5-9930-107f2e1ae09e